cloud/cloud-account.md

```diff
@@ -4,7 +4,12 @@ title: Manage your Cloud account
 sidebar_label: Cloud account
 ---
 
-Find out how to sign up for Memgraph Cloud and manage passwords.  
+Find out how to sign up for Memgraph Cloud, manage passwords and add payment method.
+
+Fell free to watch a demo video made for the Cloud launch that will explain
+the Account section of Memgraph Cloud: 
+
+[<img alt="account-payment" src={require('./data/account-payment.png').default} style={{width:'50%'}}/>](https://youtu.be/Tt5KPKylU8k?t=941 "Account section")

```

Comment:
Not a fan of multiple links/pictures from a same video source. If we have a video resource, than that resoursce should be split up in segments that maybe we could link to. But multiple images don't look so nice when there is a wide image and narrow video image etc. 

![image](https://user-images.githubusercontent.com/18125448/181749599-f41467e7-8e81-404c-ac61-f6e2897b6452.png)

Reply:
answered in slack